frontend/components/account/js/account-user-management.component.ts

```diff
@@ -41,199 +41,225 @@ const TABS = {
-		 */
-		public prepareMemberData = (member): object => {
-			return {
-				...member,
-				isAdmin: member.permissions.includes(TEAMSPACE_PERMISSIONS.admin.key),
-				isCurrentUser: this.currentUser === member.user
-			};
-		}
+	/**
+	 * Get teamspace details
+	 */
+	public onTabChange = (): void => {
+		const newParams: any = {tab: this.selectedTab};
 
-		/**
-		 * Get teamspace jobs list
-		 * @param teamspaceName
-		 */
-		public getTeamspaceJobsData(teamspaceName: string): void {
-			return this.AccountService.getJobs(teamspaceName)
-				.catch(this.DialogService.showError.bind(null, "retrieve", "jobs"))
-				.then((response) => {
-					const jobs = get(response, "data", []);
-					return {
-						jobs,
-						colors: uniq(map(jobs, "color"))
-					};
-				});
+		if (this.selectedTab !== TABS_TYPES.PROJECTS) {
+			newParams.view = null;
 		}
+		this.$state.go(this.$state.$current.name, newParams, {notify: false});
+	}
+
+	/**
+	 * Get teamspace users list
+	 * @param teamspaceName
+	 */
+	public getTeamspaceMembersData(teamspaceName: string): void {
+		const quotaInfoPromise = this.AccountService.getQuotaInfo(teamspaceName)
+			.catch(this.DialogService.showError.bind(null, "retrieve", "subscriptions"));
 
-		/**
-		 * Get teamspace projects list
-		 * @param teamspaceName
-		 */
-		public getTeamspaceProjects(teamspaceName: string): object[] {
-			if (!teamspaceName) {
-				return [];
-			}
-			// TODO: Handle request
+		const memberListPromise = this.AccountService.getMembers(teamspaceName)
+			.catch(this.DialogService.showError.bind(null, "retrieve", "members"));
+
+		return this.$q.all([quotaInfoPromise, memberListPromise])
+			.then(([quotaInfoResponse, membersResponse]) => {
+				return {
+					licencesLimit: get(quotaInfoResponse, "data.collaboratorLimit", 0),
+					members: [...membersResponse.data.members.map(this.prepareMemberData)]
+				};
+			});
+	}
+
+	/**
+	 * Convert member data to proper format
+	 * @param member
+	 * @returns
+	 */
+	public prepareMemberData = (member): object => {
+		return {
+			...member,
+			isAdmin: member.permissions.includes(TEAMSPACE_PERMISSIONS.admin.key),
+			isCurrentUser: this.currentUser === member.user
+		};
+	}
+
+	/**
+	 * Get teamspace jobs list
+	 * @param teamspaceName
+	 */
+	public getTeamspaceJobsData(teamspaceName: string): Promise<any> {
+		const jobsPromises = [
+			this.JobsService.getList(teamspaceName),
+			this.JobsService.getColors(teamspaceName)
+		];
+		return Promise.all(jobsPromises)
+			.catch(this.DialogService.showError.bind(null, "retrieve", "jobs"))
+			.then(([jobsResponse, colorsResponse]) => {
+				const jobs = get(jobsResponse, "data", []);
+				const colors = get(colorsResponse, "data", []);
+
+				return {jobs, colors};
+			});
+	}
+
+	/**
+	 * Get teamspace projects list
+	 * @param teamspaceName
+	 */
+	public getTeamspaceProjects(teamspaceName: string): object[] {
+		if (!teamspaceName) {
 			return [];
 		}
+		// TODO: Handle request
+		return [];
```

Comment:
Is this being used? It looks incomplete.

Reply:
Done ;)